plugins/image.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: image
+spec:
+  version: v0.1.0
+  homepage: https://github.com/chenjiandongx/kubectl-image
+  shortDescription: Show container images used in the cluster.
+  description: |
+    This plugin shows container images used in the Kubernetes cluster in a
+    table view. You can show all images or show images used in a specified
+    namespace.
+
+    To display a table of all images in current namespace:
+      $ kubectl image
+
+    To display a table of images in dev namespace:
+      $ kubectl image -n dev
+
+    To display a table of images in all namespaces:
+      $ kubectl image --all-namespaces
+  caveats: |
+    Usage:
+      kubectl image
+      kubectl image -n <ns>
+      kubectl image --all-namespaces
```

Comment:
not really "caveats". users should be able to discover these by -h/--help.
please remove.

Reply:
:+1: 

The caveats section is reserved for prerequisites or other important notes. Usage does not qualify for that.